src/views/Correction/FmCorrection.vue

```diff
@@ -83,7 +84,7 @@ export default class FmCorrection extends Mixins(CommonMixin, DateMixin, FilingT
 
   /** Whether this is a client error correction (vs. a staff error correction). */
   get isClientErrorCorrection (): boolean {
-    return true // FUTURE: implement this according to schema changes
+    return this.getCorrectionErrorType === CorrectionErrorTypes.CLIENT
```

Comment:
I'm thinking we should make this a store getter (since it's used in multiple places). If you're almost done with this ticket then we can do this in a future commit.

Reply:
I can make this into a store getter and add the people and roles for change of registration that you referenced above
> And also `changeofRegistration`.